kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/orders/ordering/PlacedProductOrder.java

```diff
@@ -0,0 +1,60 @@
+package com.kodilla.good.patterns.orders.ordering;
+
+import com.kodilla.good.patterns.orders.data.*;
+import java.time.LocalDate;
+
+public class PlacedProductOrder extends ProductOrder {
```

Comment:
To simplify names, I propose following changes:
- rename ProductOrder to OrderRequest
- rename PlacedProductOrder to Order

Additionally, do you remember about such rule: Favor composition over inheritance? 
What is your opinion, when code will be easier to read?

Reply:
I thougth about it. I think the case is too abstract to do the most adequate choice. But You are more experienced, so I'll change it.